### PR TITLE
Use MLIR path for plaidml2

### PR DESCRIPTION
**Pull request by flaub**
This is finally ready to land!

### CODE REVIEW

File: tile/codegen/test/tile.cc [hunk 1]
```diff
@@ -346,19 +354,6 @@ TEST(Stencil, Pass) {
   EXPECT_THAT(data["C"], ContainerEq(kMatMulExpected));
 }
 
-TEST(Codegen, TilePassBroadcast) {
```
Comment (tzerrell) [inline tile/codegen/test/tile.cc]: Are we testing broadcast elsewhere? Is broadcast temporarily broken?

File: tile/lib/lib.cc [hunk 2]
```diff
@@ -279,80 +276,90 @@ lang::RunInfo LoadConstCalc(const std::string& name) {
...
   auto K = Placeholder(kernel, "K");
-  auto runinfo = Evaluate(name, {Convolution(I, K, output)});
-  runinfo.const_inputs = {"K"};
-  runinfo.input_buffers = {{"K", MakeBuffer(kernel)}};
-  return runinfo;
+  return Program(name, {Convolution(I, K, output)});
+  // auto runinfo = Evaluate(name, {Convolution(I, K, output)});
+  // runinfo.const_inputs = {"K"};
+  // runinfo.input_buffers = {{"K", MakeBuffer(kernel)}};
+  // return runinfo;
```
Comment (tzerrell) [inline tile/lib/lib.cc]: Are the various runinfo comments in this file reminders to add `const_inputs` and `input_buffers` functionality/testing?

File: tile/codegen/test/tile.cc (hunk 1 above)
Comment (flaub) [reply tile/codegen/test/tile.cc]: This was an odd test, it didn't seem to have any assertions.

Also, I do have some broadcast tests elsewhere.

File: tile/lib/lib.cc (hunk 2 above)
Comment (flaub) [reply tile/lib/lib.cc]: I will remove these comments. The` runinfo` was mixing concerns; I have a new `Binder` object on the Execution API that should provide similar functionality. Marking constant buffers is still an open item however.

**Pull request merged**